src/Migrations/M001EnableEnFormFeature.php

```diff
@@ -19,7 +18,9 @@ class M001EnableEnFormFeature extends MigrationScript
      */
     protected static function execute(MigrationRecord $record): void
     {
-        Features\EngagingNetworks::enable();
+        $features = get_option('planet4_features');
+        $features['feature_engaging_networks'] = true;
```

Comment:
Do we need to add the `feature_engaging_networks`  index to the `planet4_features` options array only to unset it in migration 36? If so, should we add a small comment to remember why we are doing it?

Reply:
I made this adjustment just because after the changes in this PR `Features\EngagingNetworks` no longer exists. So this was raising an exception when deploying to the test instance, even if the migration has already run.